src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/dashboard/fix/data-broker-profiles/welcome-to-plus/WelcomeToPlus.test.tsx

```diff
@@ -82,6 +82,22 @@ it("shows the correct number of broker profiles with multiple broker scan result
   expect(paragraphElement).toBeInTheDocument();
 });
 
+it("checks the How-It-Works page link", () => {
+  const ComposedWelcomeToPlusView = composeStory(
+    WelcomeToPlusViewInProgressStory,
+    Meta,
+  );
+  const brokerScanCount = 5;
+  render(<ComposedWelcomeToPlusView brokerScanCount={brokerScanCount} />);
+
+  const howItWorksLink = screen.getByRole("link", {
+    name: "Removals typically take 7-14 days",
```

Comment:
```suggestion
    name: "How It Works",
```

I think the role name for this should be `"How It Works"`, If you want to check the text content you should have a reference to the element in `howItWorksLink` so if you so I think you could check e.g. `howItWorksLink.textContent === "Removals typically take 7-14 days"` if you want to check that too.